src/catalogue/items/catalogueItemsLandingPage.component.tsx

```diff
@@ -422,65 +427,69 @@ function CatalogueItemsLandingPage() {
                 )}
               </Grid>
 
-              <Grid item xs={12}>
-                <Collapse in={showManufacturer}>
-                  <Grid container spacing={1}>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <ListItem
-                        style={{
-                          justifyContent: 'flex-start',
-                          display: 'flex',
-                          flexDirection: 'row',
-                          alignItems: 'center',
-                        }}
-                      >
-                        <ListItemText
-                          primary={'Manufacturer Name'}
-                          secondary={catalogueItemIdData.manufacturer.name}
-                        />
-                      </ListItem>
-                    </Grid>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <ListItem
-                        style={{
-                          justifyContent: 'flex-start',
-                          display: 'flex',
-                          flexDirection: 'row',
-                          alignItems: 'center',
-                        }}
-                      >
-                        <ListItemText
-                          primary={'Manufacturer URL'}
-                          secondary={
-                            <MuiLink
-                              underline="hover"
-                              target="_blank"
-                              href={catalogueItemIdData.manufacturer.url}
-                            >
-                              {catalogueItemIdData.manufacturer.url}
-                            </MuiLink>
-                          }
-                        />
-                      </ListItem>
-                    </Grid>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <ListItem
-                        style={{
-                          justifyContent: 'flex-start',
-                          display: 'flex',
-                          flexDirection: 'row',
-                          alignItems: 'center',
-                        }}
-                      >
-                        <ListItemText
-                          primary={'Manufacturer Address'}
-                          secondary={catalogueItemIdData.manufacturer.address}
-                        />
-                      </ListItem>
+              {manufacturer && (
```

Comment:
Could you also modify the other ListItem's in this file above (when they don't have a parent List) please